lib/helpers/contentfulEntry.js

```diff
@@ -25,24 +39,25 @@ function getById(contentfulId) {
  * @param {Object} contentfulEntry
  * @param {Promise}
  */
-function parseContentfulEntry(contentfulEntry) {
-  // TODO: It might be nice to move all of these type of functions into this helper, to restrict
+async function parseContentfulEntry(contentfulEntry) {
+  const data = {
+    raw: contentfulEntry,
+    parsed: null,
+  };
+  // TODO: It'd be nice to move all of these type of functions into this helper, to restrict
   // lib/contentful to simply querying Contentful API.
   const contentType = contentful.getContentTypeFromContentfulEntry(contentfulEntry);
 
   if (helpers.broadcast.getContentTypes().includes(contentType)) {
-    return helpers.broadcast.parseBroadcastFromContentfulEntry(contentfulEntry);
-  }
-
-  if (helpers.topic.getContentTypes().includes(contentType)) {
-    return helpers.topic.parseTopicFromContentfulEntry(contentfulEntry);
-  }
-
-  if (helpers.defaultTopicTrigger.getContentTypes().includes(contentType)) {
-    return helpers.defaultTopicTrigger.parseDefaultTopicTriggerFromContentfulEntry(contentfulEntry);
+    data.parsed = await helpers.broadcast.parseBroadcastFromContentfulEntry(contentfulEntry);
```

Comment:
Minor suggestion. I think moving the `if/else` section to it's own function would make `parseContentfulEntry` easier to read and test.

Example:

```
// heuristic entry parser
async function getHeuristicallyParsedEntry(contentfulEntry) {
  // move heuristics here and return parsed entry ...
}


// Contentful entry response parser
async function parseContentfulEntryResponse(contentfulEntry) {
  return {
    raw: contentfulEntry,
    parsed: await getHeuristicallyParsedEntry(contentfulEntry),
  };
}
```